tests/conftest.py

```diff
@@ -136,3 +136,32 @@ def user_data():
         "id": 1
     }
     yield user_data
+
+
+@pytest.yield_fixture
+def webhook_push_data():
+    webhook_data = {
+        "webhooks": [{
+            "branch": "master",
+            "event_type": "push",
+            "host": "github.com",
+            "name": "some_repo",
+            "owner": "SomeUser",
+            "snapshots": []
+        }]
+    }
+    yield webhook_data
+
+@pytest.yield_fixture
+def webhook_release_data():
+    webhook_data = {
+        "webhooks": [{
+            "branch": "master",
```

Comment:
release hook is not per branch, you can check example [here](https://github.com/cernanalysispreservation/analysispreservation.cern.ch/blob/cd00fe3c4f2d7b276cc0d4a7df148b194382a095/tests/conftest.py#L581)